docs/user-guide/kv-config.md

```diff
@@ -137,17 +137,24 @@ traefik:
 
 NB : Be careful to give the correct IP address and port in the flag `--consul.endpoint`.
 
+## Consul ACL Token support
+
+To specify a Consul ACL token for traefik to use you must set an system environment variable
+prior to launching traefik named `CONSUL_HTTP_TOKEN` equal to the acl token value.
```

Comment:
Maybe the sentence can looks like this : `To specify a Consul ACL token for Traefik, we have to set a System Environment variable named **CONSUL_HTTP_TOKEN** prior to starting traefik. This variable must be initialized with the ACL token value.`